backend/post-service/src/main/java/com/speakapp/postservice/controllers/CommentController.java

```diff
@@ -26,6 +26,29 @@ public CommentPageGetDTO getCommentsForPost(@RequestParam(defaultValue = "0") in
         return commentService.getCommentsForPost(pageNumber, pageSize, postId, userId);
     }
 
+    @GetMapping("/by-created-at")
+    public CommentPageGetDTO getCommentsForPostByCreatedAtSorted(
+            @RequestParam(defaultValue = "0") int pageNumber,
+            @RequestParam(defaultValue = "10") int pageSize,
+            @RequestParam String sort,
+            @RequestParam UUID postId,
+            @RequestHeader("UserId") UUID userId) {
+
+        return commentService.getCommentsForPostByCreatedAt(
+                pageNumber, pageSize, postId, userId, sort);
+    }
+
+    @GetMapping("/by-reactions")
```

Comment:
wiem, że tego nie ma na jirze, ale lepiej byłoby rozwiązać to wszystko za pomocą opcjonalnych parametrów to zwykłej funkcji getowania komentarzy.

Może coś w stylu `sortByReactions: "asc" | "desc"` i `sortByCreatedAt: "asc" | "desc"`?